test/validation/api/pool/pool.c

```diff
@@ -1178,10 +1185,15 @@ static void pool_test_pool_statistics(odp_pool_type_t pool_type)
 		CU_ASSERT(stats.cache_available <= num_obj);
 		CU_ASSERT(stats.cache_alloc_ops == 0);
 		CU_ASSERT(stats.cache_free_ops == 0);
+		CU_ASSERT(stats.thread.first == first);
+		CU_ASSERT(stats.thread.last == last);
+		for (j = 0; j < ODP_POOL_MAX_THREAD_STATS; j++)
+			CU_ASSERT(stats.thread.cache_available[j] <= num_obj);
```

Comment:
CU_ASSERT(stats.thread.cache_available[j] <= stats.cache_available); would be more specific.

Reply:
Fixed in V5.